### ISSUE TITLE
Add support for Recurring Downtime

### ISSUE THREAD

**Issue by hlascelles** [DataDog/datadog-cloudformation-resources#225]
Recurring downtime is not yet supported (https://github.com/DataDog/datadog-cloudformation-resources/pull/119, https://github.com/DataDog/datadog-cloudformation-resources/pull/34, https://github.com/DataDog/datadog-cloudformation-resources/pull/21)...

But I see that it is (now?) supported by the API: https://docs.datadoghq.com/monitors/guide/suppress-alert-with-downtimes/?tab=api#recurring-downtime-on-the-nth-weekday-of-the-month

eg:

```
curl -X POST "https://api.<DATADOG_SITE>/api/v1/downtime" \
-H "Content-type: application/json" \
-H "DD-API-KEY: ${api_key}" \
-H "DD-APPLICATION-KEY: ${app_key}" \
-d '{"scope": "env:prod","start":"1613844000","end":"1613887200", "recurrence": {"type": "days","period": 1}}'
```

Does that now mean it can be added as a CF resource please?

**Comment by hlascelles** [comment]
Still valid and needed please!

**Comment by kiwi-33** [comment]
+1 it would be great if recurring downtime could added as a CFN resource.

**Comment by hlascelles** [comment]
Can we remove the `stale` label for this please? It's still a wanted feature!

**Comment by hlascelles** [comment]
Can anyone from DataDog give us an idea if this is coming? We still think it would be a great (and simple?) feature to add.

**Comment by paul110** [comment]
+1

**Comment by Alex-Sainsbury** [comment]
+1

**Comment by jjl-cdpq** [comment]
We also need this feature because we establish a standard pf using Cloudformation to configure Monitors and Downtime. Please advice when this will be implemented.

**Comment by skarimo** [comment]
Hello all, we have released 1.0.0 version of the new resource `Datadog::Monitors::DowntimeSchedule` which supports recurring downtimes: https://github.com/DataDog/datadog-cloudformation-resources/tree/master/datadog-monitors-downtimeschedule-handler#datadogmonitorsdowntimeschedule

Closing this issue as complete, thanks!
